### ISSUE TITLE
Feature Request- Dual-Channel Mono

### ISSUE THREAD

**Issue by Saijin-Naib** [GaudiLabs/OpenTheremin_V3#4]
My OpenTheremin V3 outputs mono audio over the 3.5mm jack, which makes sense. However, it only does so on one channel (Left) which makes it a bit unpleasant for headphone playing.

Is it possible to have the mono audio output on both channels, or would that negatively impact the processing load?

**Comment by Theremingenieur** [comment]
By connecting a headphone to the audio output, you risk to overload the output buffer op-amps since a headphone has an impedance of 32Ohm per side and will already in the actual mono configuration draw up to 80mA peak current which Is already too much, even with both amps in parallel. Connecting the other ear, too, will double the peak current to 160mA which that small op-amp which is designed for Lower line level currents (at loads between 600 and 50000Ohm) can not deliver. Why not build or buy a cheap LM386 based headphone amplifier module, these are 2$ on AliExpress and this chip is able to deliver the required currents even for 2 earpieces in parallel.

**Comment by Saijin-Naib** [comment]
Okay, forget headphones then:
Line-out to speaker system. Same use case: Mono is unpleasant.

Does that also present the same issue you've noted above?

**Comment by Theremingenieur** [comment]
More than 90% of electronic music instruments (theremins, guitars, basses, keyboards, synthesizers, etc.) have monaural outputs. That’s the standard. Musicians when playing alone use suitable instrument amplifiers which have also only one output channel (sometimes they have a second input which allows to add a playback track). When more “electric” musicians play together in a band, all instruments are usually connected to a huge mixing desk which allows then to virtually place them on arbitrary positions between the two main PA speakers (and to send them a mono signal back to their individual monitor speakers). That’s what the famous pan potentiometers are for.

**Comment by Saijin-Naib** [comment]
Close as wontfix?

**Comment by Theremingenieur** [comment]
It’s up to @GaudiLabs to close this. I (as a simple contributor) haven’t the appropriate access rights.

**Comment by Saijin-Naib** [comment]
> Why not build or buy a cheap LM386 based headphone amplifier module

@Theremingenieur , I hate to bother you further, but I've been having lessons with my OpenTheremin these past weeks and I've had consistent audio quality issues. Some I've reduced with better grounding, some by replacing my 12v switching power supply to my Seeeduino with a 15v linear, but still the audio issues persist.

Would something like this kit be acceptable for what you've stated above?
https://www.ebay.com/itm/264665311080

I really need to correct the audio output of my OpenTheremin so that I may continue my lessons.
Thank you for all your assistance and information!
